apps/meteor/app/markdown/lib/markdown.js

```diff
@@ -33,29 +32,11 @@ class MarkdownClass {
 	}
 
 	parseMessageNotEscaped(message) {
-		const parser = settings.get('Markdown_Parser');
-
-		if (parser === 'disabled') {
-			return message;
-		}
-
 		const options = {
-			supportSchemesForLink: settings.get('Markdown_SupportSchemesForLink'),
-			headers: settings.get('Markdown_Headers'),
 			rootUrl: Meteor.absoluteUrl(),
-			marked: {
-				gfm: settings.get('Markdown_Marked_GFM'),
-				tables: settings.get('Markdown_Marked_Tables'),
-				breaks: settings.get('Markdown_Marked_Breaks'),
-				pedantic: settings.get('Markdown_Marked_Pedantic'),
-				smartLists: settings.get('Markdown_Marked_SmartLists'),
-				smartypants: settings.get('Markdown_Marked_Smartypants'),
-			},
 		};
 
-		const parse = typeof parsers[parser] === 'function' ? parsers[parser] : parsers.original;
-
-		return parse(message, options);
+		return parsers.original(message, options);
```

Comment:
I've used the default setting value here for this function, I haven't removed because this is being used inside the autotranslate with `tokenizeCode` function